lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -102,10 +105,16 @@ def configure(instance):
         help='Root parition size, e.g. 100G',
         callback=aws_cli.convert_disk_size_to_int
     )
+    @click.option(
+        '--data',
+        required=False,
+        help='Instance data, in YAML',
+        type=click.Path(exists=True, readable=True)
```

Comment:
I would make it file, not path, then you can pass it in stdin:

--data - ... < data.yml

Reply:
OK, changed to click.File